Sources/System.swift

```diff
@@ -27,17 +27,16 @@ class System {
     static var name: String? {
         var systemInfo = utsname()
         uname(&systemInfo)
-
-        if let identifier = String(cString: &systemInfo.machine.0, encoding: .ascii) {
+        if let identifier = withUnsafePointer(to: &systemInfo.machine, {
```

Comment:
You don't need the `if` anymore: identifier is always a String with the initializer `String(cString: ptr)`

Reply:
@dmace2 Hey thanks for the PR, mind just fixing up this one as well?

Reply:
Sorry about that, been moving the last few days. Will get it done tonight!